assembly/src/parsers/tests.rs

```diff
@@ -422,13 +474,62 @@ aliqua."
 #[test]
 fn test_ast_parsing_module_docs_fail() {
     let source = "\
-    #! test message 1
+    #! module doc
+
+    #! proc doc
+    export.foo.1 
+        loc_load.0
+    end
     
-    #! test message 2
+    #! malformed doc
+    ";
+    parse_module(source).expect_err("comment message should not have empty lines");
```

Comment:
Where is this error message coming from? I couldn't find it anywhere.

Reply:
Oh, I forgot to change it. The trick is that this message is not matching with error message, it's just the string that should be printed if we get an error (unlike in integration tests, where `expect_error()` function match provided string with error message). I'll change it to the actual error message so that it doesn't confuse anymore